zboxcore/sdk/sdk.go

```diff
@@ -1015,6 +1016,23 @@ func AddCurator(curatorId, allocationId string) (string, error) {
 	return hash, err
 }
 
+func CollectRewards(poolId string, providerType int) (string, error) {
```

Comment:
Use Provider for providerType explicitly so that the sdk user can be reminded to use the Provider type from sdk instead of define it themselves.